in_text/text_02_drivers.yml

```
@@ -454,12 +469,12 @@ entities:
 
 process-date: !expr format(Sys.time(),'%Y%m%d')
 process-description: >-
-  Stream observations and driver data were combined from multiple sources to achieve the most complete temporal coverage possible
+   Stream observations and driver data were combined from multiple sources to achieve the most complete temporal coverage possible
```

The driver data gets described in a little more detail after this, but not the stream observations. Should we add some info about this?
```suggestion
  Stream observations and driver data were combined from multiple sources to achieve the most complete temporal coverage possible
```